eg/Classes/PluginInstall.py

```diff
@@ -283,3 +431,154 @@ def VisitName(self, node):
 
     def VisitStr(self, node):
         return node.s
+
+
+class HiddenSystemDialog(wx.Dialog):
+
+    def __init__(self, fileFolders):
+
+        self.fileFolders = fileFolders
+        self.root = None
+
+        wx.Dialog.__init__(
+            self,
+            None,
+            title='Plugin Install - Hidden/System Files',
+            size=(450, 525),
+            style=(
+                wx.CAPTION |
+                wx.CLOSE_BOX |
+                wx.SYSTEM_MENU |
+                wx.RESIZE_BORDER |
+                wx.MAXIMIZE_BOX
+            )
+        )
+
+        headerBox = eg.HeaderBox(
+            self,
+            name='Plugin Install',
+            text=(
+                'There are hidden and system files please select\n'
+                'the files you wish to add to the egplugin package.'
```

Comment:
Well, to me it looks like eg.classes.Text is a relict before there was eg.classes.TranslateableStrings (not sure, haven't looked into svn history). I'll admit that you're right to put the texts into the corresponding files. So we should go that way (and get rid of eg.classes.Text).